server/api/image.py

```diff
@@ -115,10 +177,12 @@ def find(self, params):
     @access.cookie
     @access.public
     def downloadMultiple(self, params):
-        Dataset = self.model('dataset', 'isic_archive')
-        File = self.model('file')
         Image = self.model('image', 'isic_archive')
 
+        include = params.get('include', 'all')
+        if include not in {'all', 'images', 'metadata'}:
+            raise RestException('Param "include" must be one of: "all", "images", "metadata"')
```

Comment:
Often we use (abuse?) `ValidationException` for invalid parameter values. Here's an example in this file: https://github.com/ImageMarkup/isic-archive/blob/c9c2e415d62eadcbcfcd368c067407b4a5112412/server/api/image.py#L348-L349. 
This could be written like:
```python
raise ValidationException('Value must be one of ...', 'include')
```

Reply:
Yeah, these rules are not really clear anywhere in Girder, but I agree that you suggestion is more consistent with other places.